README.md

```diff
@@ -1,62 +1,81 @@
 # stripe_payment
 
-A Flutter plugin to integrate the stripe plugin for iOS and Android. 
+#### Conveniently secure payments methods using Stripe.
 
-It supports:
-- Adding a credit card as payment method
-- SCA and 2FA (two factor authentication)
-- confirm + setup payments
-- Apple / Google Pay (check setup instructions on https://pub.dev/packages/stripe_native)
+## Quick Glance
 
-<img src="https://github.com/jonasbark/flutter_stripe_payment/raw/master/screenshot_android.png" width="300">
-<img src="https://github.com/jonasbark/flutter_stripe_payment/raw/master/screenshot_ios.png" width="300">
+- Collect chargable tokens from users' **Card Input** and **Apple & Google Pay**
+- For **SCA** compliant apps, setup payment intents for later confirmation.
 
-## Android
 
-**Please be aware that your main activity must extend from FlutterFragmentActivity. Otherwise the Android dialog would've looked very nasty.**
+<img src="https://github.com/jonasbark/flutter_stripe_payment/raw/master/screenshot_android.png">
 
-Include this into your project's android/gradle.properties file
+![Apple Pay](https://user-images.githubusercontent.com/7946558/65780165-02838700-e0fe-11e9-9db9-5fe4e44ed819.gif)
+
+
+## Dependencies
+
+### Android & iOS
+- Create a Stripe account and project
+- Retrieve a publishable key from the Stripe dashboard
+  
+![Stripe Dashboard](https://miro.medium.com/max/847/1*GPDsrgR6RXYuRCWiGxIF1g.png)
+
+### Android 
+- Requires AndroidX
+
+Include support in android/gradle.properties
 ```properties
 android.useAndroidX=true
 android.enableJetifier=true
 ```
 
-## Usage
+#### (Optional) Create a merchant identifier on the Google Play console
+
+### iOS (Apple Pay)
+
+1) On the Stripe dashboard, go to Settings -> Apple Pay -> Add Application.
+2) Follow the instructions that popup to link Stripe with Apple.
+
+![Stripe Apple Pay Setup](https://miro.medium.com/max/396/1*YXaDZi1VYkfNXfM9Px6dIA.png)
+
+3) Take note of your merchant identifier, and open the iOS module in Xcode.
+4) Navigate the project panel, add Apple Pay under signing capabilites with the merchant identifer created in step 1.
+
+![Apple Signing Capabilites](https://miro.medium.com/max/735/1*1-qwONvN5qMSsRYZyP57Hw.png)
+
+## Setup
+
+Using **publishable key** and **merchant identifier**, ready the app for collecting user payment information. This needs to be done prior to opening card input or a native payment sheet.
 
-To set your publishable key set:
 ```dart
-import 'package:stripe_payment/stripe_payment.dart';
-StripePayment.setSettings(StripeSettings(publishableKey: "pk_test_", merchantIdentifier: "Test", androidProductionEnvironment: false));
+var settings = StripeSettings(publishableKey:"pk_test_yZuUz6SqmH4lA7SrlAvYCh003MvJiJlR", merchantIdentifier: "merchant.stripe-example", androidProductionEnvironment: false);
```

Comment:
you wouldn't want to have that key public, right? :D

Reply:
I don't really mind as long as they cant make it to my secret key and start making charges.

I changed the ID, and key ever so slightly to not be the ones I use in the example too.